.github/workflows/self_hosted_e2e.yaml

```diff
@@ -77,25 +79,33 @@ jobs:
           podman machine start
           echo "CONTAINER_RUNTIME=podman" >> $GITHUB_ENV
       - name: Determine latest Dapr Runtime version including Pre-releases
-        if: github.base_ref == 'master'
         run: |
           export RUNTIME_VERSION=$(curl -s -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" https://api.github.com/repos/dapr/dapr/releases | grep tag_name | awk -F':' '{print $2}' | tr -d '", ' | sed '/-/! s/$/_/' | sort -V | sed 's/_$//' | tr -d 'v' | tail -1)
+          echo "DAPR_RUNTIME_LATEST_VERSION=$RUNTIME_VERSION" >> $GITHUB_ENV
           if [[ -z "$RUNTIME_VERSION" ]]; then
-            echo "Could not fetch the latest Dapr Runtime version. Using default version $DAPR_RUNTIME_VERSION"
+            echo "Could not fetch the latest Dapr Runtime version. Using default version $DAPR_RUNTIME_PINNED_VERSION"
```

Comment:
You also need to set `DAPR_RUNTIME_LATEST_VERSION` to `DAPR_RUNTIME_PINNED_VERSION`, right?

Reply:
No, not needed. These two will be used separately to track the pinned(directly set via `env` in workflow files) and latest versions.

Reply:
What will this imply then "Using default version $DAPR_RUNTIME_PINNED_VERSION"?
The idea here was that if the cURL request fails due to GitHub API being down or throttling, `$DAPR_RUNTIME_LATEST_VERSION` will still contain a sane value instead of empty.

Reply:
Corrected.